config/locales/views/emails/en.yml

```diff
@@ -0,0 +1,9 @@
+---
+en:
+  emails:
+    all:
+      title: All
+    inbox:
+      title: Inbox
+    reply_later:
+      title: Reply later  
```

Comment:
This is great as well, but how are we using these at the moment?

Reply:
Sorry it's not visible in the diff, if you look at the content of the following file:
app/views/emails/all.html.erb
app/views/emails/inbox.html.erb
app/views/emails/reply_later.html.erb

There is `<%= t('.title')%>` in there. This is where the title is referenced. This was previously done in `config/locales/views/pages/en.yml`

Hope that answers?

Reply:
Ah yes I see! Thanks for providing more info.